pkg/component/component.go

```diff
@@ -1372,3 +1365,21 @@ func getStorageFromConfig(localConfig *config.LocalConfigInfo) storage.StorageLi
 	}
 	return storageList
 }
+
+// checkIfURLChangesWillBeMade checks to see if there are going to be any changes
+// to the URLs when deploying and returns a true / false
+func checkIfURLChangesWillBeMade(client *occlient.Client, componentConfig config.LocalConfigInfo) (bool, error) {
```

Comment:
I am assuming you want a test for this one here @girishramnani

Reply:
Yes

Reply:
This is not exported function. Not sure if it is that important to have this covered.

`ApplyConfig` that is calling this should be definitely covered and have test cases for this. (Doesn't have to be part of this PR)

@girishramnani @cdrage The decision is up to you.